apps/web/src/app/globals.css

```
@@ -818,34 +818,60 @@
-  --timeline-reveal-x: 0px;
   opacity: 0;
+  transition: opacity 500ms cubic-bezier(0.22, 1, 0.36, 1);
+}
+
+.timeline-reveal [data-timeline-item].timeline-item--entered {
+  opacity: 1;
+}
+
+.timeline-reveal [data-timeline-card] {
+  --timeline-reveal-x: 0px;
   transform: translateX(var(--timeline-reveal-x)) translateY(24px);
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)